test/common/shared.spells.test.js

```diff
@@ -0,0 +1,128 @@
+/* eslint-disable camelcase */
+let shared = require('../../common/script/index.js');
+
+let newUser = (addTasks = true) => {
+  let buffs;
+  let user;
+
+  buffs = {
+    per: 0,
+    int: 0,
+    con: 0,
+    str: 0,
+    stealth: 0,
+    streaks: false,
+  };
+  user = {
```

Comment:
It is to my understanding that most of this is needed during this test. When commenting out the data to find out what I did not need, I ran into errors. I will get you some more specific details soon.